### PR TITLE
[Tech#362] switch component

### PR DESCRIPTION
**Pull request by aycil-alican**
switch component was added to UIKit list with generic classes that was created by @robergro 

### Pull requests template

This pull request has:
- [ ] Code documentation on publics
- [ ] Accessibility
	- [ ] Dynamic sizes
	- [ ] Identifiers
- [ ] Tests
- [ ] Demo update
- [ ] Wiki

### CODE REVIEW

File: spark/Demo/Classes/View/Components/Switch/UIKit/SwitchComponentUIView.swift [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        }
+
+        self.viewModel.$textContent.subscribe(in: &self.cancellables) { [weak self] content in
+            guard let self = self else { return }
+
+            self.viewModel.contentConfigurationItemViewModel.buttonTitle = content.name
+
+            switch content {
+            case .text:
+                self.componentView.attributedText = nil
```
Comment (robergro) [inline spark/Demo/Classes/View/Components/Switch/UIKit/SwitchComponentUIView.swift:74]: Not needed, you just set the text

File: spark/Demo/Classes/View/Components/Switch/UIKit/SwitchComponentUIView.swift [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+                self.componentView.text = viewModel.text
+
+            case .attributedText:
+                self.componentView.text = nil
```
Comment (robergro) [inline spark/Demo/Classes/View/Components/Switch/UIKit/SwitchComponentUIView.swift:78]: Not needed, you just set the attributed text

File: spark/Demo/Classes/View/Components/Switch/UIKit/SwitchComponentUIView.swift [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                self.componentView.attributedText = viewModel.attributedText
+
+            case .multilineText:
+                self.componentView.attributedText = nil
```
Comment (robergro) [inline spark/Demo/Classes/View/Components/Switch/UIKit/SwitchComponentUIView.swift:82]: Not needed, you just set the text

Comment (aycil-alican) [reply spark/Demo/Classes/View/Components/Switch/UIKit/SwitchComponentUIView.swift]: fixed

**Review by robergro — APPROVED** [review]

**Pull request merged**